app/constants/constants.js

```diff
@@ -0,0 +1,6 @@
+export const LOCATION_DATA = 'LOCATION_DATA';
```

Comment:
`constants/constants.js` is vague enough that any constant can go here. and that means this file will quickly become dumping ground of every constant.

suggest `constants/storage.js`